client/my-sites/stats/videopress-stats-module/index.jsx

```diff
@@ -151,93 +155,171 @@ class VideoPressStatsModule extends Component {
 			page( url );
 		};
 
+		const isNewVideoPage = config.isEnabled( 'stats/new-video-summary' );
+
+		const downloadCSV = (
+			<div className="stats-module__heaver-nav-button">
```

Comment:
I need to fix the breakpoint for this to only go under the title for small screens.

Reply:
Actually, I won't be fixing this because the period selection will replace the button like on the other summary pages.